upload-api/test/ucan-invocation.test.js

```diff
@@ -443,7 +443,8 @@ test('can process ucan log request for given receipt after its invocation stored
   const message = await CAR.request.decode(workflow)
 
   const invocationCid = message.invocations[0].cid
-  t.deepEqual(invocationCid, invocation.link())
+
+  t.deepEqual(invocationCid.bytes, invocation.link().bytes)
```

Comment:
this was to work around some very slight mismatch with a new property `[Symbol.toStringTag]` for CID. still trying to figure out where that comes from, but I figure this is still a pretty good deepEqual for a cid right?